torchdata/datapipes/iter/load/aisio.py

```diff
@@ -98,14 +98,14 @@ class AISFileLoaderIterDataPipe(IterDataPipe[Tuple[str, StreamWrapper]]):
 
     Example:
         >>> from torchdata.datapipes.iter import IterableWrapper, AISFileLister,AISFileLoader
-        >>> ais_prefixes = IterableWrapper(['ais://bucket-name/folder/', 'aws:bucket-name/folder/', ...])
+        >>> ais_prefixes = IterableWrapper(['gcp://bucket-name/folder/', 'aws:bucket-name/folder/', ...])
```

Comment:
Curious about why we want to change it from `aid` to `gcp`? Why not we still keep `ais` but append a `gcp` to show case for users.

Reply:
yep, good idea. I wanted to show that GCP can be used because none of the current data pipes were doing that. I will add `ais` and keep `gcp`

Reply:
Do you mind doing it for `Lister` as well?

Reply:
done